packages/application-components/src/components/public-page-layout/public-page-layout.styles.ts

```diff
@@ -0,0 +1,680 @@
+import styled from '@emotion/styled';
+import { designTokens as uiKitDesignTokens } from '@commercetools-uikit/design-system';
+
+export const Container = styled.div`
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  padding: ${uiKitDesignTokens.spacingXl} 0;
+  justify-content: center;
+  background-size: cover;
+  background-position: center;
+`;
+
+export const ContainerColumn = styled.div`
+  width: calc(${uiKitDesignTokens.constraint16} / 2);
+`;
+export const ContainerColumnWide = styled.div`
+  width: ${uiKitDesignTokens.constraint15};
+`;
+
+export const GradientBackgroundContainer = styled.div`
+display: flex;
+justify-content: center;
+align-items: center;
+height: 100vh;
+width: 100%;
+
+--1x: 25%;
+--1y: 25%;
+--2x: 75%;
+--2y: 25%;
+--3x: 25%;
+--3y: 75%;
+--4x: 75%;
+--4y: 75%;
+background: radial-gradient(
+    70% 70% at var(--3x) var(--3y),
+    var(--color-primary-90) 1%,
+    transparent 65%
+  ),
+  radial-gradient(
+    70% 70% at var(--2x) var(--2y),
+    var(--color-primary-90) 1%,
+    transparent 75%
+  ),
+  radial-gradient(
+    30% 90% at var(--4x) var(--4y),
+    var(--color-turquoise-90) 0% 5%,
+    transparent 80%
+  ),
+  radial-gradient(
+    30% 90% at var(--1x) var(--1y),
+    var(--color-turquoise-90) 0% 5%,
+    transparent 80%
+  );
+background-blend-mode: color;
+animation: gradients 10s ease both infinite;
```

Comment:
It looks like the keyframes definition isn’t being recognized by the browser (see attached screenshot).
<img width="840" alt="Screenshot 2024-08-30 at 09 31 12" src="https://github.com/user-attachments/assets/2e29f167-274e-4055-8b0d-52b6f5c6e2b2">

Maybe we could try moving the `@keyframes gradient` definition higher up in the styled component to ensure it gets picked up correctly?

Reply:
good catch, thank you Kacper!